tests/integration/modules/test_vm.py

```diff
@@ -39,103 +39,140 @@ def test_vm_list(integration_test_config, patch_salt_globals_vm):
     assert all == integration_test_config["virtual_machines"]
 
 
-def test_vm_list_templates(integration_test_config, patch_salt_globals_vm):
-    """
-    Test vm list_templates
-    """
-    all = virtual_machine.list_templates()
-    assert all == integration_test_config["virtual_machines_templates"]
-
-
-def test_ovf_deploy(integration_test_config, patch_salt_globals_vm):
-    """
-    Test deploy virtual machine through an OVF
-    """
-    res = virtual_machine.deploy_ovf(
-        vm_name="test1",
-        host_name=integration_test_config["esxi_host_name"],
-        ovf_path="tests/test_files/centos-7-tools.ovf",
-    )
-    assert res["deployed"] == True
-
-
-def test_ova_deploy(integration_test_config, patch_salt_globals_vm):
-    """
-    Test deploy virtual machine through an OVA
-    """
-    tar = tarfile.open("tests/test_files/sample.tar", "w")
-    tar.add("tests/test_files/centos-7-tools.ovf")
-    tar.close()
-    res = virtual_machine.deploy_ova(
-        vm_name="test2",
-        host_name=integration_test_config["esxi_host_name"],
-        ova_path="tests/test_files/sample.tar",
-    )
-    os.remove("tests/test_files/sample.tar")
-    assert res["deployed"] == True
-
-
-def test_template_deploy(integration_test_config, patch_salt_globals_vm):
-    """
-    Test deploy virtual machine through an template
-    """
-    if integration_test_config["virtual_machines_templates"]:
-        res = virtual_machine.deploy_template(
-            vm_name="test_template_vm",
-            template_name=integration_test_config["virtual_machines_templates"][0],
-            host_name=integration_test_config["esxi_host_name"],
-        )
-        assert res["deployed"] == True
-    else:
-        pass
-
-
-def test_path(integration_test_config, patch_salt_globals_vm):
-    """
-    Test deploy virtual machine through an template
-    """
-    if integration_test_config["virtual_machine_paths"].items:
-        for k, v in integration_test_config["virtual_machine_paths"].items():
-            res = virtual_machine.path(vm_name=k)
-            assert res == v
-    else:
-        pass
-
-
-def test_powered_on(integration_test_config, patch_salt_globals_vm):
-    """
-    Test deploy virtual machine through an template
+# def test_vm_list_templates(integration_test_config, patch_salt_globals_vm):
+#     """
+#     Test vm list_templates
+#     """
+#     all = virtual_machine.list_templates()
+#     assert all == integration_test_config["virtual_machines_templates"]
```

Comment:
**question** why are these commented out? If they're dead/useless tests, let's remove them. If there's a reason they should be skipped, use `@pytest.skip('reason')` (might be `.mark.skipIf(True, 'reason')`)

Reply:
Oh jeez, this is a full mistake. They were commented so I didn't have to run all of them while I was creating the new tests.

Reply:
FWIW, next time just use `-k 'some_test'` to run the ones that you're interested in. Or `-k 'not some_other'` to skip the ones you're disinterested in.

Or `@pytest.mark.skipIf(True, 'TODO just testing, bring these back in')`